packages/calypso-e2e/src/lib/pages/woocommerce-landing-page.ts

```diff
@@ -1,6 +1,6 @@
 import { Page } from 'playwright';
 
-export const selectors = {
```

Comment:
Tidy up from https://github.com/Automattic/wp-calypso/pull/59548, this shouldn't have been exported, gets exposed by `lib/pages/index.ts`'s `export * from './woocommerce-landing-page';`